ci/travis/deps.sh

```diff
@@ -28,17 +43,7 @@ deps_dragonboard() {
     sudo apt-get -qq update
 	sudo service docker restart
     sudo docker pull rycus86/arm64v8-debian-qemu
-
-    # Clone glog now so we don't need git inside of
-    # the docker container
-    pushd "${DEPS_DIR}"
-    [ -d "glog" ] || {
-       git clone https://github.com/google/glog
-    }
-    pushd glog
-    git checkout tags/v0.3.5
-    popd
-    popd
 }
 
+get_deps_source_code
```

Comment:
Shouldn't "get_deps_source_code" be called only in "deps_default()" and "deps_dragonboard()". There is no reason to get the source code for the dependencies when running builds that do: cppcheck, clang-format, doxygen.

Reply:
You are right. Thanks!